### DISCUSSION TITLE
Filecoin Docs. Change Requests/Ideas/Discussion

### DISCUSSION THREAD

**Discussion by cwhiggins** [filecoin-project/lotus#5479] [Storage Provider] [answered]
I am starting to take a look at the Filecoin Docs, there are some things that can be updated, and maybe some things that should be changed.  First area I am looking at is the Max Fee section of the lotusminer config.toml.  Current Docs show default MaxWindowPoStGasFee=50FIL, does anyone know if this is still the default value? I am set at 1.5FIL and have not had issues. I am looking for the default values created after init for the miner config.toml, if any one can point me to them, that would be helpful.

**Comment by ribasushi** [comment, marked as answer]
A general mechanism to answering such questions is to just lean on github's search. Specifically in your case: https://github.com/filecoin-project/lotus/search?q=MaxWindowPoStGasFee

Cheers!

**Reply by cwhiggins** [reply]
Awesome, I searched config.toml, this is much better. I will figure it out

**Comment by jennijuju** [comment]
Thanks for catching this! For docs requests, please submit here https://github.com/filecoin-project/filecoin-docs/issues !

**Comment by rjan90** [comment]
One idea that I have had for either a Filecoin Docs, or as a blog post, is a "Best Practices When Running a Miner" that encompasses things around security, backups, maintenance of disk, networking, etc.

It could just be a bulleted list of things (with links to articles about that topic) that you should keep in mind / do as a miner, that will set you better up in case of/when things breaks, or gets destroyed.

It would also be a very convenient article/docs for new miners joining, that we can point to.
